src/Tools/Test Framework/Test Runner/src/DataDrivenTest/DataInputs/TestInputGroup.Table.al

```diff
@@ -45,6 +45,12 @@ table 130454 "Test Input Group"
             CalcFormula = count("Test Input" where("Test Input Group Code" = field(Code)));
             ToolTip = 'Specifies the number of entries in the dataset.';
         }
+        field(60; "Imported by AppId"; Guid)
```

Comment:
I would say that here we need an MD5 so we do not override the definition if it is not needed.

Reply:
Yes, that could be an optimization, the current design is:
1. Overwrite dataset if it is imported by the same test app
2. Throw an error if the app is modifying an existing dataset uploaded by a different app

We can me it same as test suite import logic.

Reply:
I was using ["Cryptography Management"](https://learn.microsoft.com/en-us/dynamics365/business-central/application/system-application/codeunit/system.security.encryption.cryptography-management) which is in the system app. AL Test Tool does not have the dependency to the system app